Libs/MRML/Core/vtkMRMLSegmentationStorageNode.cxx

```diff
@@ -388,14 +379,14 @@ int vtkMRMLSegmentationStorageNode::ReadBinaryLabelmapRepresentation4DSpatial(vt
           currentSegmentName);
     currentSegment->SetName(currentSegmentName.c_str());
 
-    // DefaultColor
-    std::string defaultColorValue;
+    // Color
+    std::string colorString;
```

Comment:
What happens if the color is not specified? Can we look up a default from terminology?